pkgs/misc/emulators/mgba/default.nix

```diff
@@ -59,9 +48,21 @@ in stdenv.mkDerivation rec {
     qttools
   ];
 
-  postInstall = ''
-    cp -r ${desktopItem}/share/applications $out/share
-  '';
+  postInstall = let
+    desktopItem = makeDesktopItem {
+      name = "mgba";
+      exec = "mgba-qt";
+      icon = "mgba";
+      comment = "A Game Boy Advance Emulator";
+      desktopName = "mgba";
+      genericName = "Game Boy Advance Emulator";
+      categories = "Game;Emulator;";
+      startupNotify = "false";
+    };
+  in
+    ''
+      cp -r ${desktopItem}/share/applications $out/share
+    '';
```

Comment:
Why not use copyDesktopItems?

Reply:
Legacy code. I didn't remember to update this at the time.